src/olympia/devhub/templates/devhub/versions/edit.html

```diff
@@ -123,7 +123,7 @@ <h3>{{ title }}</h3>
           {% with field = version_form.source %}
             <th>
               <label for="{{ field.auto_id }}">{{ _("Source code") }}</label>
-              {{ tip(None, _("If your add-on contain binary or obfuscated code, make the source available here for reviewers.")) }}
+              {{ tip(None, _("If your add-on contain minified, concatenated or otherwise machine-generated code, make the source available here for reviewers.")) }}
```

Comment:
While we're here, we might as well fix this to say "If your add-on contains", rather than "If your add-on contain".